httpx/met.go

```diff
@@ -0,0 +1,36 @@
+package httpx
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/bsm/rucksack/met"
+	"github.com/go-chi/chi/middleware"
+)
+
+var metMiddleware = middleware.RequestLogger(metFormat{})
+
+type metFormat struct{}
+
+func (metFormat) NewLogEntry(r *http.Request) middleware.LogEntry { return metEntry{Request: r} }
+
+type metEntry struct{ *http.Request }
+
+func (e metEntry) Write(status, _ int, elapsed time.Duration) {
```

Comment:
Not sure if very useful, but sometimes it's interesting to check the response size (I mean, let's met `bytes` param?)

Though this is really minor and can be added later (even enabled via ENV).

Reply:
yeah, next version :)